bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/obsrecord/VisitCalculator.scala

```diff
@@ -129,9 +135,14 @@ private[obsrecord] object VisitCalculator {
         instrument.exists(_.isVisitor)
 
       def hasChargeableDataset: Boolean =
-        dsets.exists { case (lab, _) => qa(lab).isChargeable }
+        dsets.exists { case (lab, _) => datasetQa(lab).isChargeable }
 
-      if (isVisitor || hasChargeableDataset) normalCharges
+      // REL-3628: many GPI acquisitions don't produce datasets so charge for
+      //           them regardless
+      def isGpiAcquisition: Boolean =
```

Comment:
I suppose this is due to the A&G steps that don't produce a dataset and are of class Acquisiton